app/components/github_integration/mentions/integration.py

```diff
@@ -31,6 +39,36 @@ class MentionActions(ItemActions):
     action_plural = "mentioned these entities"
 
 
+@tasks.loop(hours=1)
+async def update_recent_mentions() -> None:
+    mention_linker.free_dangling_links()
+    entity_to_message_map: defaultdict[Entity, list[dc.Message]] = defaultdict(list)
```

Comment:
```suggestion
    entity_to_message_map = defaultdict[Entity, list[dc.Message]]()
```
Though this does mean `Entity` has to be taken out of the type-checking block.